ShurikenMC/Infected-Factions
# add i18n for great good

**Summary**

this commit adds in i18n for any user-facing command output. right
now this is only configurable on a server-level. it might be useful
one-day to have a per-user configuration. however that would add
quite a bit of memory overhead which for an 0.0.1 release this isn't
really worthwhile.

**Test Plan**

- `mvn verify` works
- Load up a server with translation files, and your locale set to `en` in config.yml
  run `/if_create test` note how all text are in english.
- Change locale to `es` in config.yml, and reboot the server.
  run `/if_create test` note how all text is spanish
